apps/web/src/styles/theme/drawer.ts

```diff
@@ -0,0 +1,24 @@
+import { drawerAnatomy as parts } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+
+import colors from "../colors";
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(parts.keys);
```

Comment:
do we need all the styles copied from the desktop? I'd rather have just the theme.ts file and then fill it in as we progress with the UI implementation. WDYT?

Reply:
Totally agree